airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -140,8 +137,9 @@ public void importDataWithSeed(String targetVersion, File archive, ConfigPersist
       // 3. Import Postgres content
       importDatabaseFromArchive(sourceRoot, targetVersion);
 
-      // 4. Import Configs
-      importConfigsFromArchive(sourceRoot, seedPersistence, false);
+      // 4. Import Configs and update connector definitions
+      importConfigsFromArchive(sourceRoot, false);
+      configRepository.loadData(seedPersistence);
```

Comment:
Maybe I'm missing something, but doesn't `loadData` need to handle the upgrade checks that we were handling in `importConfigsFromArchive` before to ensure we aren't overwriting source and destination definitions?

Reply:
Yes, `DatabaseConfigPersistence#loadData` does avoid updating a connector if it is used. The `FileSystemConfigPersistence#loadData` overwrites everything, but it is not used. Let me change it to throw an exception.

These existing tests ensure that we don’t update a connector definition if it is in use:
https://github.com/airbytehq/airbyte/blob/master/airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/DatabaseConfigPersistenceLoadDataTest.java
https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/test/java/io/airbyte/server/handlers/ArchiveHandlerTest.java#L163